packages/lxml/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: lxml
+  version: 4.4.1
+source:
+  sha256: c81cb40bff373ab7a7446d6bbca0190bccc5be3448b47b51d729e37799bb5692
+  url: https://files.pythonhosted.org/packages/c4/43/3f1e7d742e2a7925be180b6af5e0f67d38de2f37560365ac1a0b9a04c015/lxml-4.4.1.tar.gz
+build:
+  cflags: |
+    -I../../../../libxml/libxml2-2.9.10/include
+    -I../../../../libxslt/libxslt-1.1.33/
+    -L../../../../zlib/zlib-1.2.11/include
+    -I../../../../libiconv/libiconv-1.16/include
+    -Wno-implicit-function-declaration
+  ldflags: |
+    -L../../../../libxml/libxml2-2.9.10/.libs/
+    -L../../../../libxslt/libxslt-1.1.33/libxslt/.libs/
+    -L../../../../libxslt/libxslt-1.1.33/libexslt/.libs/
+    -L../../../../zlib/zlib-1.2.11/lib/
+    -L../../../../libiconv/libiconv-1.16/lib/.libs/
```

Comment:
Do you understand why they don't need to link to libiconv in conda-forge https://github.com/conda-forge/lxml-feedstock/blob/master/recipe/meta.yaml while it's done here?

Reply:
Looks like libiconf is not really needed and I will remove, but when I try to compile without zlib I get this warning:
`shared:WARNING: emcc: cannot find library "z"`

I think lxml is not using the static version of `libxml2` and is asking for its dependencies on link.

Do you know how can I "force" these libraries to link statically? Or is it not needed?

Reply:
> when I try to compile without zlib I get this warning:
shared:WARNING: emcc: cannot find library "z"

Do you still get it if you add `-s USE_ZLIB=1` to `cflags` as in https://github.com/iodide-project/pyodide/blob/12a42508fc841130796a3e73f3bd5042351ebf08/packages/matplotlib/meta.yaml#L26 ?

Reply:
> Do you still get it if you add -s USE_ZLIB=1 to cflags as in 

Yes.

meta.yml:
```
build:
  cflags: |
    -s USE_ZLIB=1
    -I../../../../libxml/libxml2-2.9.10/include
    -I../../../../libxslt/libxslt-1.1.33/
    -Wno-implicit-function-declaration
```

warning context:
```
Installed /src/cpython/build/3.7.4/host/lib/python3.7/site-packages/lxml-4.4.1-py3.7-linux-x86_64.egg
Processing dependencies for lxml==4.4.1
Finished processing dependencies for lxml==4.4.1
emcc -s USE_ZLIB=1 -I../../../../libxml/libxml2-2.9.10/include -I../../../../libxslt/libxslt-1.1.33/ -Wno-implicit-function-declaration -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -Isrc -Isrc/lxml/includes -I/src/cpython/installs/python-3.7.4/include/python3.7 -c src/lxml/etree.c -o build/temp.linux-x86_64-3.7/src/lxml/etree.bc -w
emcc -O3 -s BINARYEN_METHOD=native-wasm -Werror -s EMULATED_FUNCTION_POINTERS=1 -s EMULATE_FUNCTION_POINTER_CASTS=1 -s SIDE_MODULE=1 -s WASM=1 -s BINARYEN_TRAP_MODE=clamp --memory-init-file 0 -s LINKABLE=1 -s EXPORT_ALL=1 -s ERROR_ON_MISSING_LIBRARIES=0 -L../../../../libxml/libxml2-2.9.10/.libs/ -L../../../../libxslt/libxslt-1.1.33/libxslt/.libs/ -L../../../../libxslt/libxslt-1.1.33/libexslt/.libs/ -pthread -shared build/temp.linux-x86_64-3.7/src/lxml/etree.bc -lxslt -lexslt -lxml2 -lrt -lz -lm -o build/lib.linux-x86_64-3.7/lxml/etree.cpython-37m-x86_64-linux-gnu.wasm
shared:WARNING: emcc: cannot find library "z"
```